content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -100,7 +100,7 @@ Pod Template:
   Labels:  tier=frontend
   Containers:
    php-redis:
-    Image:        gcr.io/google_samples/gb-frontend:v3
+    Image:        us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v3
```

Comment:
```suggestion
    Image:        us-docker.pkg.dev/google-samples/containers/gke/gb-frontend:v5
```

Reply:
Only one tag v5 exists and tested v3 will also not work as not available in repository.